silverstripe/graphql/CVE-2023-40180.yaml

```diff
@@ -0,0 +1,20 @@
+title:     "CVE-2023-40180 DDOS Vulnerability on GraphQL due to lack of protection against recursive queries"
+link:      https://www.silverstripe.org/download/security-releases/CVE-2023-40180
+cve:       CVE-2023-40180
+branches:
+    3.8.x:
+        time:     2023-10-16 00:44:54
+        versions: ['>=3.0.0', '<3.8.2']
```

Comment:
There are different branches for 3.0 to 3.7. Would it make sense to list each of them?

Reply:
I have listed those in a single set based on this discussion from last time I submitted a pr:
https://github.com/FriendsOfPHP/security-advisories/pull/688#discussion_r1177539786